packages/gestalt/src/ErrorMessage.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Box from './Box.js';
+import formatErrorId from './formatErrorId.js';
+import Text from './Text.js';
+
+type Props = {|
+  errorMessage?: string,
+  id: string,
+|};
+
+export default function ErrorMessage({ errorMessage, id }: Props) {
+  return errorMessage ? (
+    <Box marginTop={1}>
+      <Text color="orange">
+        <span id={formatErrorId(id)}>{errorMessage}</span>
+      </Text>
+    </Box>
+  ) : null;
```

Comment:
Is this what design wanted? rendering `null` when there is no error will cause jitters in the UI when the error appears because itll occupy new space, that feels off to me

Reply:
I think I was being overly defensive, and I don't feel strongly about it. I'll get rid of the ternary.